src/generators.rs

```diff
@@ -1732,14 +1719,13 @@ pub fn dorogovtsev_goltsev_mendes_graph(py: Python, n: usize) -> PyResult<graph:
 /// (interactions). Each node represents a club member, and each edge represents
 /// a relationship between two members.
 ///
-/// :param bool multigraph: When set to ``False`` the output
+/// :param bool multigraph: When set to ``False``, the output
 ///     :class:`~rustworkx.PyGraph` object will not be not be a multigraph and
 ///     won't allow parallel edges to be added. Instead
 ///     calls which would create a parallel edge will update the existing edge.
 ///
 /// :returns: The undirected graph representing the Karate club graph.
```

Comment:
I don't find this Karate club graph on https://www.rustworkx.org/api/generators.html